credmark/cmf/types/token.py

```diff
@@ -122,26 +100,43 @@ def info(self):
 
         return TokenInfo(**self.dict(), meta=self._meta)
 
+    def try_erc20_property(self, prop_name):
```

Comment:
This could actually be MORE generic than Token, this could be a contract thing if there's other Contract standards, like Proxies, or 721s or whatever.